client/packages/programs/src/JsonForms/common/components/Select.tsx

```diff
@@ -168,22 +185,78 @@ const getHighlightParts = (
   );
 };
 
+/**
+ * Returns either the full list or the narrowed list from conditional option.
+ *
+ * If, after applying the filter condition, the currentSelection is not in the filtered list
+ * the currentSelection item is added to the returned list and an error message is returned.
+ */
```

Comment:
```suggestion
/**
 * Returns either the full list or the narrowed list from conditional option.
 *
 * If, after applying the filter condition, the currentSelection is not in the
 * filtered list the currentSelection item is added to the returned list and an
 * error message is returned.
 */
```

The line wrapping is weird on mine -- this is how my comment auto-formatter breaks it up.